ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -139,7 +158,20 @@ public struct Avatar: View, TokenizedControlView {
         let presenceCutoutOriginX: CGFloat = layoutDirection == .rightToLeft ? presenceCutoutOriginXRTL : presenceCutoutOriginXLTR
         let presenceCutoutOriginY = presenceCutoutOriginXLTR
         let presenceIconFrameSideRelativeToOuterRing: CGFloat = presenceIconFrameSideRelativeToInnerRing + outerGapAndRingThicknesCombined
-        let overallFrameSide = max(ringOuterGapSize, presenceIconFrameSideRelativeToOuterRing)
+        let presenceOverallFrameSide = max(ringOuterGapSize, presenceIconFrameSideRelativeToOuterRing)
+
+        let activityCutoutOriginXLTR: CGFloat = AvatarTokenSet.avatarSize(size) - activityIconOutlineSize + activityBorderThickness + ringInnerGap + outerGapAndRingThicknesCombined
+        let activityCutoutOriginXRTL: CGFloat = 0 - activityBorderThickness + ringInnerGap + outerGapAndRingThicknesCombined
+        let activityCutoutOriginX: CGFloat = layoutDirection == .rightToLeft ? activityCutoutOriginXRTL : activityCutoutOriginXLTR
```

Comment:
should large portion of these calculation be part of the custom modifier so we don't necesarily have to do this calculation unless we actually need to calculate activity cutout?